orchagent/aclorch.cpp

```diff
@@ -386,6 +387,16 @@ bool AclRule::validateAddMatch(string attr_name, string attr_value)
         return false;
     }
 
+    // NOTE: Temporary workaround to support matching protocol numbers on MLNX platform.
+    // In a later SAI version we will transition to using NEXT_HEADER for IPv6 on all platforms.
+    string platform = getenv("platform") ? getenv("platform") : "";
```

Comment:
>getenv [](start = 43, length = 6)

You called `getenv` twice? Could you do once?

Reply:
I see one more in AclOrch::init()

---
In reply to: [450405491](https://github.com/Azure/sonic-swss/pull/1343#discussion_r450405491) [](ancestors = 450405491)

Reply:
Yeah that's where I got it from! I will fix the additions for this PR and then go back and change it for the existing code in a follow-up (because I think it's done in a few more places as well ).